os/fs/driver/mtd/smart.c

```diff
@@ -307,7 +308,9 @@ struct smart_struct_s {
 	size_t journal_seq;				/* Current Sequence of Journal */
 	uint8_t journalPerSector;		/* Number of journal data per Sector */
 	uint16_t total_journalsector;	/* Total Number of Journal Sector */
+	FAR char *jbuf;					/* Journal Data buffer */
 	FAR uint16_t *sSeqLogMap;		/* Keep duplicated sector until journaling determine winner */
+	int journal_area_state[2];		/* State of Journal Area */
```

Comment:
in my opinion, It seems better to add a definition for 2.

Reply:
Thanks for comment, this is draft version to share with amogh actually =)

AnywayI explained in KConfig file regarding journal area, (MTD_SMART_JOURNALING_NSECTOR).
User should read when enable it. We have 2 area to toggle active area.
Active area being used to save journal data, this never changed, kind of absolute value.

Reply:
yes, i got it.